docs/tutorial/marsjob.md

```diff
@@ -0,0 +1,225 @@
+# Run Mars workloads on kubernetes
+
+## What's Mars
+
+`Mars` is a tensor-based unified framework for large-scale data computation which scales Numpy, Pandas and Scikit-learn, 
+see [mars-repo](https://github.com/mars-project/mars) for details. As a data computation framework, `mars` is easy to 
+scale out and can run across hundreds of machines simultaneously to boost processing large scale data tasks.<br>
+
+A distributed mars job includes 3 roles to collaborate with each other：
+- **WebService**: web-service accepts requests from end-users and forwards the whole tensor-graph to scheduler, it provides a dashboard with end users so that they can track job status and submit tasks interactively.
```

Comment:
provides a dashboard for end users to track...

Reply:
done